### PR TITLE
Update landing page for NumPy chapter

### LINKED ISSUE #78: NumPy chapter landing page needs updating
As discussed in #39, the [landing page for the NumPy chapter](https://foundations.projectpythia.org/core/numpy.html) is still bare and needs updating.

### PR DESCRIPTION
**Pull request by brian-rose**
Closes #78

### CODE REVIEW

**Comment by brian-rose** [comment]
This is very basic, but better than a blank page!

File: core/numpy.md [hunk 1]
```diff
@@ -4,4 +4,12 @@
...
 ```
 
-This section will contain tutorials on array computing with [NumPy](https://numpy.org).
+This section contains tutorials on array computing with [NumPy](https://numpy.org).
+
+---
+
+From the [NumPy documentation](https://numpy.org/doc/stable/user/whatisnumpy.html):
+
+> NumPy is the fundamental package for scientific computing in Python. It is a Python library that provides a multidimensional array object, various derived objects (such as masked arrays and matrices), and an assortment of routines for fast operations on arrays, including mathematical, logical, shape manipulation, sorting, selecting, I/O, discrete Fourier transforms, basic linear algebra, basic statistical operations, random simulation and much more.
```
Comment (jukent) [inline core/numpy.md]: I think we want a `,` between "random simulation" and "and much more. I can add that and then approve. This looks great.

**Review by jukent — APPROVED** [review]

File: core/numpy.md (hunk 1 above)
Comment (brian-rose) [reply core/numpy.md]: That's totally fine. I would have written it that way myself (I'm a fan of the Oxford comma), but this is actually a direct copy-and-paste quote from the Numpy docs.

No big deal either way.

**Comment by brian-rose** [comment]
This seems non-controversial enough to merge after one approval!

**Pull request merged**